### PR TITLE
Some more fixes for building docs

### PR DESCRIPTION
**Pull request by lucc**
Sorry there where quite some things I overlooked in #912.

### CODE REVIEW

File: docs/source/configuration/accounts_table [hunk 1]
```diff
@@ -47,7 +57,7 @@
 
      where to store outgoing mails, e.g. `maildir:///home/you/mail/Sent`.
      You can use mbox, maildir, mh, babyl and mmdf in the protocol part of the URL.
-
+
```
Comment (pazz) [inline docs/source/configuration/accounts_table]: Nitpick: should we try to avoid these trailing whitespaces to be introduced?

Comment (lucc) [inline docs/source/configuration/accounts_table]: Done.

**Comment by lucc** [comment]
I used two different approaches to remove the trailing ws from the autogen. docs. Do you think this is ok or would you prefer to remove trailing ws in the second script as well (I think that might be a bit more difficult as with the first script).

**Comment by lucc** [comment]
Also NB: The "Changes approved" mark from your review was not removed by github even though I pushed new commits.

**Comment by pazz** [comment]
re reviews: yes, that's strange.. next time i'll simply hold back longer :)
I think the current way of dealing with trailing spaces is ok. Its not the end of the world anyway if one sneaks in after all..

**Pull request merged**